pkg/apis/build/v1alpha2/builder_types.go

```diff
@@ -36,7 +36,8 @@ type BuilderSpec struct {
 	Stack corev1.ObjectReference `json:"stack,omitempty"`
 	Store corev1.ObjectReference `json:"store,omitempty"`
 	// +listType
-	Order []BuilderOrderEntry `json:"order,omitempty"`
+	Order            []BuilderOrderEntry `json:"order,omitempty"`
+	AdditionalLabels map[string]string   `json:"additionalLabels,omitempty"`
```

Comment:
Can you regenerate the CRDs as well?

Reply:
I just added a new commit.

Please let me know if you think it's not the right place